sklearn/discriminant_analysis.py

```diff
@@ -93,7 +123,14 @@ def _class_means(X, y):
     return means
 
 
-def _class_cov(X, y, priors, shrinkage=None):
+def _class_cov(
+        X,
+        y,
+        priors,
+        shrinkage=None,
+        covariance_estimator=None,
+        standardize=False
+):
```

Comment:
it'd be nice if you follow the conventions like the rest of the codebase, in this case it'd be:

```python
def _class_cov(X, y, priors, shrinkage=None, covariance_estimator=None,
        standardize=False):
```